src/player.cpp

```diff
@@ -833,25 +834,36 @@ DepotChest* Player::getDepotChest(uint32_t depotId, bool autoCreate)
 		return nullptr;
 	}
 
-	it = depotChests.emplace(depotId, new DepotChest(ITEM_DEPOT)).first;
+	uint16_t depotItemId = getDepotBoxId(depotId);
+	if (depotItemId == 0) {
```

Comment:
```suggestion
if (!depotItemId) {
```

Reply:
No, this is checked explicitly, because it returns a number, not a boolean nor pointer.